Tutorials/01_Psi4NumPy-Basics/1g_basis-sets.ipynb

```diff
@@ -163,7 +163,7 @@
     "eb, wb = psi4.energy('scf', return_wfn=True)\n",
     "psi4.compare_strings(\"c2v\", bzb.schoenflies_symbol(), \"Point group\")                       \n",
     "psi4.compare_values(refnuc, bzb.nuclear_repulsion_energy(), 10, \"Nuclear repulsion energy\")\n",
-    "psi4.compare_values(refscf, eb, 10, \"SCF Energy\")                                  "
+    "psi4.compare_values(refscf, eb, 9, \"SCF Energy\")                                  "
```

Comment:
Generally anything more than ~7 is pretty strict.

Reply:
I changed it to 9 because I knew that would make it pass, should I loosen it further to 7?

Reply:
It would be a good policy to do so.